Monika After Story/game/styles.rpy

```diff
@@ -388,3 +390,148 @@ init -10 python in mas_ui:
             if data["return_value"] == data["true_value"]:
                 return selected_prompt
         return default_prompt
+
+    # Methods for twopane_scrollable_menu
+    def _twopane_menu_filter_events(ev, search_query, search_kws, only_pool, only_random, only_unseen, only_seen):
+        """
+        A method to use as the filter for events in the twopane menu
+
+        IN:
+            ev - event object
+            search_query - search query to filter by
+            search_kws - search_query splitted using spaces
+
+        OUT:
+            boolean whether or not the event pass the criteria
+        """
+        ev_label = ev.eventlabel.lower()
+        ev_prompt = ev.prompt.lower()
+        ev_prompt_kws = ev_prompt.split()
+
+        return (
+            ev_prompt != ev_label
+            and ev.unlocked
+            # and (ev.pool or ev.random)
+            and not ev.anyflags(store.EV_FLAG_HFNAS)
+            and ev.checkAffection(store.mas_curr_affection)
+            and (
+                (not only_pool or ev.pool)
+                and (not only_random or ev.random)
+                and (not only_unseen or ev.shown_count == 0)
+                and (not only_seen or ev.shown_count > 0)
+            )
+            and (
+                search_query == ev_prompt
+                or search_query == ev_label
+                or search_query in ev_prompt
+                or search_query in ev_label
```

Comment:
`x in y` also covers `x == y` so could remove the `==` checks here

Reply:
Difference: `"you" in "your"` and `"you" == "your"` (first returns `True`, second `False`). I use both to get more precise results.

Reply:
I think I see what you mean. In `filter`, it may be enough to just do `in` check, it's in `sort` where we need to do both. I'll test and change that if I can.